docs/cave/d2d-gis-shapefiles.md

```diff
@@ -99,16 +102,18 @@ Selecting the **Sort** option opens the **Sort Order** dialog.
 
 Within the Sort Order dialog, you can select a column from the dropdown list of displayed columns and select how you want the information in that column sorted. The other columns will adjust accordingly. Clicking the OK button (hidden in the Exhibit under the dropdown list) closes the Sort Order dialog and saves the change according to your selection. Clicking the Cancel button closes the dialog without making any changes to the Attributes Table.
 
+## Labeling GIS Data
 
-##  Displaying a Sample of GIS Data Attribute Values\
-
-You can view a sample of a selected GIS data attribute directly on the Main Display Pane without having to open the Attributes dialog.
+You can select which attribute you want to use to label the objects on the Main Display.
 
-To open the Select Sample Attribute submenu, click and hold B3 on the Map Product ID in the Legend area to open a pop-up menu and select the Select Sample Attribute option. On the Select Sample Attribute submenu select the value you want displayed and then click the OK button. Click and hold B3 anywhere on the Main Display Pane to open a pop-up menu and check the Sample checkbox to activate the sampling functionality. Then, hovering your cursor over any area of the Map Product image will display the sample for the selected attribute
+To open the Label submenu, click and hold B3 on the Map Product ID in the Legend area to open a pop-up menu and select the Label and choose which attribute you want as the label.
 
+![image](../images/gis-label.png)
 
+<!--
 ##  GIS Data and Display Customization from within the GIS Application
```

Comment:
I didn't think this needed since it's like any other products. I've just commented this out for now.